com.gitlab.cunidev.Gestures.json

```diff
@@ -0,0 +1,41 @@
+{
+    "app-id" : "com.gitlab.cunidev.Gestures",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "40",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "gestures",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--filesystem=~/.config",
+        "--filesystem=xdg-run/gvfs",
```

Comment:
What is it needed for?

Reply:
If you mean gvfs, for importing/exporting "profile files", that is, backing up the configuration to some file and reloading in case of need.

Reply:
Using GtkFileChooserNative will let you drop it as it delegates filesystem access to a portal.

Reply:
I am already using GtkFileChooserNative, but without it I was getting warnings about it ("falling back on session-bus"). I will retry taking it away